sqitch_pg/plan.sh

```diff
@@ -0,0 +1,36 @@
+pkg_name=sqitch_pg
+pkg_origin=core
+pkg_version="3.7.4"
+pkg_maintainer="The Habitat Maintainers humans@habitat.sh"
+pkg_license=('Artistic-1.0-Perl' 'GPL-2.0')
+pkg_deps=(
+  core/glibc
+  core/perl
+  core/postgresql
+  core/zlib
+  core/sqitch
+)
+pkg_build_deps=(
+  core/cpanminus
+  core/local-lib
+  core/gcc
+  core/make
+)
+pkg_description="Sqitch the database management application, bundled with the DBD::Pg Perl module for PostgreSQL"
+pkg_upstream_url="http://search.cpan.org/dist/DBD-Pg/"
```

Comment:
I could go either way on this.   do you link to the actual thing being installed by this package (DBD::Pg) or to the thing you're wrapping?   I chose the former, but am happy to switch to the latter

Reply:
Good question - this is something we have definitely not be consistent about in core plans.  Should we perhaps consider starting an RFC so that we can be consistent on it?  I think this is fine for now.

Reply:
Our [documentation](https://www.habitat.sh/docs/reference/#pkg_upstream_url) specifies that the pkg_upstream_url is expected to be "An upstream project homepage or website URL. ", so  I think linking to the sqitch pgsql doc might be better in this case.

Reply:
good point @nellshamrell - changing